rmf_traffic/test/unit/test_Trajectory.cpp

```diff
@@ -242,67 +242,67 @@ SCENARIO("Segment Unit Tests")
     inputs.push_back({time + 20s, UnitBox, Eigen::Vector3d(2, 2, 2), Eigen::Vector3d(0, 0, 0)});
     rmf_traffic::Trajectory trajectory = create_test_trajectory(inputs);
     rmf_traffic::Trajectory::iterator trajectory_it = trajectory.begin();
-    rmf_traffic::Trajectory::Segment &segment = *trajectory_it;
-    rmf_traffic::Trajectory::Segment &segment_10s = *(++trajectory_it);
+    rmf_traffic::Trajectory::Waypoint &waypoint = *trajectory_it;
+    rmf_traffic::Trajectory::Waypoint &segment_10s = *(++trajectory_it);
 
     WHEN("Setting a new profile using set_profile function")
```

Comment:
This test is also no longer relevant.
```suggestion
    WHEN("Setting a new profile using set_profile function")
```